src/modules/dashboard/components/HookedTokenIcon/HookedTokenIcon.tsx

```diff
@@ -44,11 +45,21 @@ const loadTokenImages = async (address: Address): Promise<Response> => {
   return fetch(tokenImageUrl);
 };
 
+const getBase64image = (blob): Promise<any> => {
```

Comment:
I think this should be an util function, as we might use it for the user / colony avatar as well